source/proof-of-concept-guide/poc-vulnerability-detection.rst

```diff
@@ -51,12 +51,65 @@ The Vulnerability Detection module is enabled by default. You can perform the fo
 
       $ sudo systemctl restart wazuh-manager
 
-Visualize the alerts
---------------------
+Visualizing the alerts
+======================
 
-You can visualize the alert data in the Wazuh dashboard. To do this, go to the Vulnerability Detection module, select Events, and click on any vulnerability to expand the document.
+To access information about vulnerabilities detected in your system, follow these steps:
+
+1. **Navigate to the Thread Intelligence Section**: In the Wazuh dashboard, locate the "Thread Intelligence" section. This is where you'll find insights into potential threats and vulnerabilities.
+
+2. **Find the Vulnerability Detection Module**: Within the Thread Intelligence section, look for the "Vulnerability Detection" module. Click on it to access detailed information about vulnerabilities across all endpoints.
+
+.. thumbnail:: /images/poc/vulnerabilities-section.png
+   :title: Detected vulnerability detection section.
+   :align: center
+   :width: 80%
+
+Dashboard
+---------
+
+This dashboard offers valuable insights into your system's vulnerabilities. It includes:
+
+- **Summary of Top 5 Vulnerabilities**: Quickly identify the most critical vulnerabilities that require immediate attention.
+  
+- **Distribution of Vulnerabilities by Severity Levels**: Understand the severity distribution of vulnerabilities, helping you prioritize your response effectively.
+
+- **Total Number of Vulnerabilities**: Get an overview of the total number of vulnerabilities grouped by their severity levels.
+
+.. thumbnail:: /images/poc/vulnerabilities-dashboard.png
+   :title: Vulnerability detection dashboard.
+   :align: center
+   :width: 80%
+
+
+Inventory
+---------
+
+The Inventory feature empowers users to search all indexed vulnerabilities using Elastic search capabilities. It offers:
+
+- **Advanced Search Functionality**: Utilize Elastic search to perform detailed searches across all indexed vulnerabilities. With flexible filtering options, you can narrow down your search based on various criteria such as severity, affected software, or vulnerability type.
+
+- **Custom Filters**: Tailor your search using filters of different types, enabling you to refine your results and focus on specific aspects of vulnerabilities that matter most to your organization. Whether it's filtering by CVE ID, affected operating system, or patch status, the Inventory feature provides the flexibility you need to find relevant information efficiently.
+
+- **Detailed Package Information**: Gain access to comprehensive details of vulnerable packages, including version numbers, dependencies, and associated CVEs. This allows you to assess the impact of vulnerabilities accurately and make informed decisions regarding patching and remediation strategies.
+
+.. thumbnail:: /images/poc/vulnerabilities-inventory.png
+   :title: Vulnerability detection inventory.
+   :align: center
+   :width: 80%
+
+Events
```

Comment:
Add a note explaining that this works with events after the first scan that works as a baseline

Reply:
Done!